Makefile

```diff
@@ -251,7 +250,8 @@ docker.exporter: build.exporter Dockerfile.exporter
 .PHONY: deps
 deps: header
 	@echo '--> Resolving dependencies...'
-	dep ensure
+	go mod tidy 
+	go mod download
```

Comment:
If we do a `go mod download`, the dependencies will be downloaded to `$GOPATH/pkg/mod`. But for building we will still be using the vendor directory right? Can we fix that.